Journal of Aerospace Information Systems/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains/Blockchain Properties for Near-Planetary, Interplanetary and Metaplanetary Space Domains.tex

```diff
@@ -175,13 +175,13 @@ \section{Literature Review}\label{sec:litreview}
 
 Properties of the near-Earth orbital space environment that impose additional limitations on blockchain technologies include:
 \begin{itemize}
-\item Limited computing system resources. Limitations include computational capabilities and memory\cite{CCSDS_2007} and storage\cite{yu_blockchain_2019}. Electrical power available to be dedicated to computation is also generally limited, as is the ability of a spacecraft thermal control system to maintain a computing system within operational limits.
-\item ``environmental restrictions including noisy, bandwidth limited, asymmetrical, and interrupted communications links''\cite{CCSDS_2007}.
+\item \textit{computing system resources}. Limitations include computational capabilities, memory\cite{CCSDS_2007} and storage\cite{yu_blockchain_2019}. Electrical power available to be dedicated to computation is also generally limited, as is the ability of a spacecraft thermal control system to maintain a computing system within operational limits.
+\item \textit{environmental conditions.} Limiting environmental conditions include ``noisy, bandwidth limited, asymmetrical, and interrupted communications links''\cite{CCSDS_2007}.
 \end{itemize}
 
 Communication delays may be compounded by mission-specific requirements. For example, some space missions include ``a requirement for early access to transferred data regardless of its quality''\cite{CCSDS_2007}. Choices between early, low-quality data and later higher-quality data, coupled with ever-changing relative positions and thus propagation delays, possibilities for communications blackouts due to positions, power availability, errors, or other factors combine to make specific situations unique.
 
-The farther from Earth one operates in space, the more propagation delays affect communications. Such delays are an important facet of space systems design\cite{CCSDS_2007,yu_blockchain_2019}. Delays are a particularly important design criterion for blockchain consensus algorithms because timeouts are a very common feature of such algorithms to determine when consensus cannot occur, and to denote error conditions.
+The farther from Earth one operates in space, the more propagation delays affect communications. Such delays are an important facet of space systems design\cite{CCSDS_2007,yu_blockchain_2019}. Delays are a particularly important design criterion for blockchain consensus algorithms because timeouts are key in determining when consensus cannot occur, and to denote error conditions.
```

Comment:
Your suggestion steps on one of Roberto's comments. His point was that not all existing consensus algorithms rely on timeouts. Can you think of a better way to rephrase while maintaining his comment?

Reply:
1) Delays are a particularly important design criterion for blockchain consensus algorithms because timeouts are often key in determining when consensus cannot occur, and to denote error conditions. OR
2) Delays are a particularly important design criterion for blockchain consensus algorithms because timeouts are often necessary to determine when consensus cannot occur, and to denote error conditions.
Or whatever you prefer. I am just not sure about the use of the word 'feature' when talking about timeouts.